src/index.ts

```diff
@@ -34,432 +35,409 @@ export const startVercel = async (req: VercelRequest, res: VercelResponse) => {
+  };
+  const buttons = Object.keys(buttonOptions).map((key) =>
+    Markup.button.callback(buttonOptions[key], key),
+  );
+  ctx.reply(prompt, Markup.inlineKeyboard(buttons));
+});
 
 /*
-  NEW -> ANSWERING FIRST ORDER
+  NEW -> ENTERING_EMAIL
 */
-bot.action("new.quickstart", async ctx =>{
-  const prompt = "Which of the following is NOT a DEX?"
-  const buttonOptions: {[k: string]: string }= {
-    "answering-first-order.1": "Jupiter",
-    "answering-first-order.2": "Raydium",
-    "answering-first-order.3": "Orca",
-    "answering-first-order.4": "Phoenix",
-  }
-  const buttons = Object.keys(buttonOptions).map(key => Markup.button.callback(buttonOptions[key], key))
-  ctx.reply(prompt, Markup.inlineKeyboard(buttons))
+
+bot.action('new.email-info', async (ctx) => {
+  const prompt = `Chomp Bot uses your email to create a new Solana wallet for you to play\\. Your email address will be the owner and sole custodian of the wallet\\. [Learn more](https://gator\\.fyi)\\.
+*Please respond with your email address to continue\\.*`;
+  ctx.replyWithMarkdownV2(prompt);
 });
 
 /*
-  ANSWERING FIRST ORDER -> ANSWERING SECOND ORDER
+  Email Message received
 */
-const secondPrompt = "What percentage of people do you think answered Raydium?"
-const secondButtonOptions: {[k: string]: string }= {
-  "answering-second-order.0": "0%",
-  "answering-second-order.10": "10%",
-  "answering-second-order.20": "20%",
-  "answering-second-order.30": "30%",
-  "answering-second-order.40": "40%",
-  "answering-second-order.50": "50%",
-  "answering-second-order.60": "60%",
-  "answering-second-order.70": "70%",
-  "answering-second-order.80": "80%",
-  "answering-second-order.90": "90%",
-  "answering-second-order.100": "100%",
-}
-const secondButtons = Object.keys(secondButtonOptions).map(key => Markup.button.callback(secondButtonOptions[key], key))
-const formattedSecondButtons = [
-  secondButtons.slice(0, 5),  // Elements 0-4
-  secondButtons.slice(5, 6),  // Element 5
-  secondButtons.slice(6)      // Elements 6 and beyond
-];
-
-bot.action("answering-first-order.1", async ctx =>{
-  ctx.reply(secondPrompt, Markup.inlineKeyboard(formattedSecondButtons))
-})
+// https://stackoverflow.com/questions/201323/how-can-i-validate-an-email-address-using-a-regular-expression
+const emailRegex =
+  /(?:[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*|"(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21\x23-\x5b\x5d-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])*")@(?:(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?|\[(?:(?:(2(5[0-5]|[0-4][0-9])|1[0-9][0-9]|[1-9]?[0-9]))\.){3}(?:(2(5[0-5]|[0-4][0-9])|1[0-9][0-9]|[1-9]?[0-9])|[a-z0-9-]*[a-z0-9]:(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21-\x5a\x53-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])+)\])/;
 
-bot.action("answering-first-order.2", async ctx =>{
-  ctx.reply(secondPrompt, Markup.inlineKeyboard(formattedSecondButtons))
-})
+bot.hears(emailRegex, async (ctx) => {
+  const { match: emailAddresses } = ctx;
```

Comment:
Shouldn't let the user do this if they have already registered an email.

Reply:
I validate it before initializing email verification. Fallback mechanism is implemented on both email and OTP hear events so that if user enters similar to that regex, it will act accordingly.

> https://github.com/gator-labs/chomp-bot/blob/111461fb51f32d4e84c1accbd15f1d9f2654865f/src/index.ts#L356-L366

and 

> https://github.com/gator-labs/chomp-bot/blob/111461fb51f32d4e84c1accbd15f1d9f2654865f/src/index.ts#L407-L412